modules/monitoring-listing-acting-alerting-rules.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
+// * monitoring/managing-alerts.adoc
 
 [id="listing-acting-alerting-rules_{context}"]
 = Listing acting alerting rules
```

Comment:
What do you mean by "acting"? And what are the differences between the last few section in this doc where you describe how to access alerts (+ rules) from the UI?

Reply:
The 'Configuring Alertmanager' section and its subheadings (including 'Listing acting alerting rules') will be replaced with the following two sections that we discussed in the 4.6 planning doc:

- Sending notification to an external system (only admin atm)
- Hi mr admin, you can fully customize Alertmanager by doing X

This first PR is only a restructuring of existing content, so I kept these as is initially and will develop what we discussed into a subsequent PR. Sorry I should have made that clearer. We decided in the docs team that it would be easier to to an initial restructuring PR then build new content and the flow on top of that, rather than submitting a giant PR with all changes.

Reply:
Makes sense.